src/typings/Types.ts

```diff
@@ -92,40 +92,44 @@ export type SurroundingPairName =
   | SimpleSurroundingPairName
   | ComplexSurroundingPairName;
 
-export type ScopeType =
-  | "argumentOrParameter"
-  | "anonymousFunction"
-  | "attribute"
-  | "class"
-  | "className"
-  | "collectionItem"
-  | "collectionKey"
-  | "comment"
-  | "functionCall"
-  | "functionName"
-  | "ifStatement"
-  | "list"
-  | "map"
-  | "name"
-  | "namedFunction"
-  | "regularExpression"
-  | "statement"
-  | "string"
-  | "type"
-  | "value"
-  | "condition"
-  | "section"
-  | "sectionLevelOne"
-  | "sectionLevelTwo"
-  | "sectionLevelThree"
-  | "sectionLevelFour"
-  | "sectionLevelFive"
-  | "sectionLevelSix"
-  | "selector"
-  | "xmlBothTags"
-  | "xmlElement"
-  | "xmlEndTag"
-  | "xmlStartTag";
+// TODO: Move to constants file;
+export const scopes = [
```

Comment:
I'd actually still call this 
```suggestion
export const scopeTypes = [
```

Because a scope type is a type of scope, where as a scope is a specific instance of that scope type. 

But open to a terminology discussion if that seems odd / confusing